chunk/chunk.go

```diff
@@ -290,6 +291,11 @@ func NewValidatorStore(store Store, validators ...Validator) (s *ValidatorStore)
 	}
 }
 
+// SetPssCallback allows injecting a callback func on the ValidatorStore struct
+func (s *ValidatorStore) SetPssCallback(f func(Chunk) error) {
```

Comment:
best to name using the `with` convention. 

```golang
func (s *ValidatorStore) WithDeliverCallback(f func(Chunk)) *ValidatorStore
```

Reply:
done